recipes/mne-realtime/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "mne-realtime" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7eb763dc75ad45572b7d3a7e26b08503ece6bcc9bdea82d8dde95b8ca339ad23
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

Just a cosmetic change really.

---

I took this straight from the template on `main`, maybe this should get updated too :)

---

Ahh ok thanks for the heads up. By the way, the easiest way to generate packages is with Grayskull: https://github.com/conda-incubator/grayskull 
It also formats everything correctly and in 95% of all cases creates a perfect package.